index.php

```diff
@@ -102,102 +102,121 @@ function gtag(){dataLayer.push(arguments);}
-                      <!-- <div class="count_top usd">
-                        <?php // $fee = round($gpRecs2['fastest']*21000/1e9*$exchangeRate/10, 3); echo($currString . $fee . ' / Transfer'); ?>
-                      </div> -->
-                    </div>
-                  </div>
-                </div>
-
-                <div class="col-md-4 col-sm-4 col-xs-6 tile_stats_count">
-                  <div>
-                    <div class="count standard">
-                      <?php echo ($gpRecs2['fast']/10) ?>
-                    </div>
-                    <div class="divider"></div>
-                    <div class="text-container">
-                      <div class="count_top"><span>fast</span><span>< 2m</span></div>
-                      <!-- <div class="count_top usd">
-                        <?php // $fee = round($gpRecs2['fast']*21000/1e9*$exchangeRate/10, 3); echo($currString . $fee . ' / Transfer'); ?>
-                      </div> -->
-                    </div>
-                  </div>
-                </div>
-
-                <div class="col-md-4 col-sm-4 col-xs-6 tile_stats_count">
-                  <div>
-                    <div class="count safe_low" id="medTx">
-                      <?php echo ($gpRecs2['average']/10) ?>
-                    </div>
-                    <div class="divider"></div>
-                    <div class="text-container">
-                      <div class="count_top"><span>standard</span><span>< 5m</span></div>
-                      <!-- <div class="count_top usd">
-                        <?php // $fee = round($gpRecs2['average']*21000/1e9*$exchangeRate/10, 3); echo($currString . $fee . ' / Transfer'); ?>
-                      </div> -->
-                    </div>
-                  </div>
-                </div>
-
-                <div class="<?php echo (isset($_GET['eip1559']) ? "show" : "hidden") ?>">
-                  <div class="top_tiles_title">
-                    Recommended (<a href="https://ethgasstation.info/blog/eip-1559/">EIP-1559</a>) tipping
-                  </div>
-
+                <div class="legacy-tx-type">
+                  <div class="top_tiles_title">Recommended Gas Prices in Gwei</div>
                   <div class="col-md-4 col-sm-4 col-xs-6 tile_stats_count">
                     <div>
                       <div class="count fast">
-                        <?php echo ($gpTip['instant']) ?>
+                        <?php echo ($gpRecs2['fastest']/10) ?>
                       </div>
                       <div class="divider"></div>
                       <div class="text-container">
                         <div class="count_top"><span>trader</span><span>< ASAP</span></div>
+                        <!-- <div class="count_top usd">
+                          <?php // $fee = round($gpRecs2['fastest']*21000/1e9*$exchangeRate/10, 3); echo($currString . $fee . ' / Transfer'); ?>
+                        </div> -->
```

Comment:
Do we want to keep these commented out code or should we simply drop it?

Reply:
Cleanup up these comments and also changed things to display the base fee in gwei. Rebased as well for cleaner commits